### ISSUE TITLE
Question about the pt_encoder

### ISSUE THREAD

**Issue by XinhanWang** [yangaonidaye123/NRCL#1]
Hello! While reviewing your code repository, I noticed that the forward pass of the point cloud encoder in NRCL seems to differ from the implementations in [Cross-Modal-Center-Loss](https://github.com/LongLong-Jing/Cross-Modal-Center-Loss/blob/5025837561a5e059940462edde99aafe80680871/models/dgcnn.py#L148-L149) and [RONO](https://github.com/penghu-cs/RONO/blob/978d92887ebb8bbb36a240a6e019d4f4902869d4/models/feat_dgcnn.py#L24-L26).

For reference:

NRCL: https://github.com/yangaonidaye123/NRCL/blob/bf4b4f339472d3ca200004ca9e02882b6271319b/models/pt_encoder.py#L18-L22
CMCL and RONO:
```Python
    def forward(self, x):
        x = F.relu(self.bn6(self.linear1(x)))
        return x
```
Could you kindly clarify the reason for this difference? Thank you very much!

Best regards!

**Comment by yangaonidaye123** [comment]
Thanks very much for your attention.  We made a slight adjustment to the network when extending our framework to image–text datasets for cross-modal learning, so that the architecture better fits the characteristics of image–text feature distributions.
It is important to note that for the 2D–3D retrieval experiments, we strictly followed the original baseline architecture to ensure fair comparison and full reproducibility. In addition, when the models are reverted to the baseline design, all results reported in the paper remain fully reproducible. 

At the same time, we observed that directly using the pt_encoders from the open-source project and using the pt_encoders reconstructed based on the baselines yield almost identical results, with negligible differences.A possible reason is that reducing the feature dimensionality from 512 to 256 typically has only a minor impact on performance due to redundancy in the original feature space.

**Comment by XinhanWang** [comment]
Thank you very much for your detailed explanation. It completely clarifies my confusion regarding the implementation differences. I appreciate your work!
